### PR TITLE
Fix formatting job on windows

### LINKED ISSUE #69053: Formatting jobs failing on Windows after native toolset upgrade
It seems that after #67771 the Windows formatting job is failing on every PR (e.g. #69041, #69051). The log shows:
```scala
Formatting jit directory.
Neither compile_commands.json exists, nor is there a build log. Running CMake to generate compile_commands.json.
Running: D:\a\1\s\src\coreclr\build-runtime.cmd x64 Checked -configureonly -ninja
BUILD: Starting Build at  7:49:12.22
BUILD: Error: Visual Studio 2019 or 2022 with C++ tools required. Please see https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/windows-requirements.md for build requirements.
There was an error running CMake to generate compile_commands.json. Please do a full build to generate a build log.
```

### PR DESCRIPTION
**Pull request by jakobbotsch**
Fix #69053

### CODE REVIEW

**Comment by jakobbotsch** [comment]
@kunalspathak PTAL, cc @dotnet/jit-contrib

**Review by kunalspathak — APPROVED** [review]
LGTM

File: eng/pipelines/coreclr/templates/format-job.yml [hunk 1]
```diff
@@ -46,18 +58,8 @@ jobs:
...
-        addToPath: true 
-        architecture: 'x64'
-      condition: ${{ eq(parameters.osGroup, 'windows') }}
-    - task: PythonScript@0
-      displayName: Run tests/scripts/format.py
-      inputs:
-        scriptSource: 'filePath'
-        scriptPath: $(Build.SourcesDirectory)/src/tests/Common/scripts/format.py
-        arguments: '-c $(Build.SourcesDirectory)/src/coreclr -o $(osGroup) -a $(archType)'
+    - script: $(PythonScript) $(Build.SourcesDirectory)/src/tests/Common/scripts/format.py -c $(Build.SourcesDirectory)/src/coreclr -o $(osGroup) -a $(archType)
```
Comment (hoyosjs) [inline eng/pipelines/coreclr/templates/format-job.yml:61]: Are you guaranteed python is installed in all the images we use?

**Review by hoyosjs — APPROVED** [review]

**Comment by hoyosjs** [comment]
Merging to unblock PRs

File: eng/pipelines/coreclr/templates/format-job.yml (hunk 1 above)
Comment (jakobbotsch) [reply eng/pipelines/coreclr/templates/format-job.yml:61]: Not sure, but we use the same approach in the rest of our JIT runs that use Python scripts and haven't run into issues yet.

**Pull request merged**